cxopen.py

```diff
@@ -0,0 +1,11 @@
+# cross platform support for opening files (for example images)
+import sys
+
+
+def get_command_for_open():
```

Comment:
forgot to mention this before, the type checker isn't happy (`mypy --strict .`), it wants every function to have a type definition.

Reply:
Ah sorry, that is also addressed now. I am working on getting the pre-commit installed (had a bit of trouble with my nix installation of Python, I think I found a solution).